packages/cli/src/commands/deploy.ts

```diff
@@ -228,6 +228,10 @@ export const action: WorkspaceCommandAction<DeployArgs> = async ({
     }
   )
   outputLine(postDeployActionsOutput.join('\n'), output)
+  const deploymentUrl = _.get(result.extraProperties?.deploymentUrls, 0)
+  if (deploymentUrl !== undefined) {
+    outputLine(`You can see your deployment here: ${deploymentUrl}`, output)
+  }
```

Comment:
```suggestion
  const deploymentUrls = result.extraProperties?.deploymentUrls
  if (deploymentUrls !== undefined) {
    outputLine(`You can see your deployment here:\n${deploymentUrls.join('\n')}`, output)
  }
```
no reason to limit this to just one url if we support multiple urls...

Reply:
v